src/Sylius/Bundle/UiBundle/Resources/config/menu.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services
+                               http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <parameters>
+        <parameter key="sylius.menu_builder.class">Sylius\Bundle\UiBundle\Menu\MenuBuilder</parameter>
```

Comment:
You're right, you can delete the <code>parameters</code> block and <code>class</code> of the service definition because you pass a new class in the <code>sylius.menu_builder.admin.main</code> service definition